src/llmcompressor/transformers/finetune/text_generation.py

```diff
@@ -370,6 +381,22 @@ def main(
     if training_args.do_predict:
         stage_runner.predict()
 
+    # wrap model.save_pretrained
+    model = trainer.model
+    if is_fsdp_model(model):
+        modify_fsdp_model_save_pretrained(trainer, tokenizer)
+    else:
+        modify_save_pretrained(model)
+
+    # save if model was provided as a string or custom output_dir was set
+    if isinstance(model_args.model, str) or (
+        training_args.output_dir
+        != TrainingArguments.__dataclass_fields__["output_dir"].default
+    ):
+        model.save_pretrained(training_args.output_dir)
```

Comment:
Could you confirm if this saves the recipe as well?

Reply:
yes

Reply:
yes its saved